backend/products/models.py

```diff
@@ -57,3 +59,26 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return self.title
+
+
+class ProductReview(models.Model):
+    product = models.ForeignKey(
+        Product, on_delete=models.CASCADE, related_name="reviews"
+    )
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="reviews"
+    )
+    rating = models.PositiveIntegerField(
+        validators=[MinValueValidator(1), MaxValueValidator(5)]
+    )
+    comment = models.TextField(blank=True, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        unique_together = ("product", "user")
```

Comment:
In Django ≥ 3.2, unique_together is deprecated in favor of UniqueConstraint in Meta.constraints.
Here is my suggested changes:
```suggestion
class Meta:
    constraints = [
        models.UniqueConstraint(fields=["product", "user"], name="unique_product_review")
    ]
```